lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -31,7 +31,7 @@ function addToUsersBucket(canonicalID, bucketName, bucketMD, log, cb) {
             constants.splitter : constants.oldSplitter;
         let key = createKeyForUserBucket(canonicalID, splitter, bucketName);
         const omVal = {
-            creationDate: new Date().toJSON(),
+            creationDate: new Date(bucketMD.getCreationDate()).toJSON(),
```

Comment:
can we just use `bucketMD.getCreationDate()` ?

Reply:
Setting the date to the bucket creation date may be OK for the "first" account, but would not work if the bucket is added to multiple accounts.

Maybe that is not supported at the moment, but as a general "design" approach it would be better to avoid relying on correlations : this usersBucket supports could work with multiple accounts 'sharing' a bucket, so let's not break it; and the precise information on the creation date of a bucket is precisely in the metastore, so we should get it from there...

As you mentioned this "fix" does not work for existing buckets, which reading the creating date from metastore does...